platform/ios/src/MGLMapView.mm

```diff
@@ -1948,8 +1949,9 @@ - (void)showAttribution
     
     UIViewController *viewController = self.window.rootViewController;
     if ([viewController isKindOfClass:[UINavigationController class]]) {
-        viewController = [(UINavigationController *)viewController viewControllers].firstObject;
+        viewController = ((UINavigationController *)viewController).topViewController;
     }
+    viewController = [UIViewController getTopViewControllerFrom:viewController];
```

Comment:
Sorry, missed this in between some work on https://github.com/mapbox/mapbox-gl-native/pull/8837/commits/4903db64fde813ff2f5ecbc313f877e580791572 @boundsj. Basically, that's what I was already working on 😄.